packages/libs/react-components/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+// TODO: Remove this when this issue is resolved: https://github.com/kadena-community/kadena.js/issues/201
+
+import { SystemIcons } from '../Icons';
+
+import {
+  StyledHeading,
+  StyledIconContainer,
+  StyledNotification,
+  StyledText,
+} from './styles';
+
+import { VariantProps } from '@stitches/react';
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: (typeof SystemIcons)[keyof typeof SystemIcons];
+  title: string;
+  children?: React.ReactNode;
+  displayCloseButton?: boolean;
+  expand?: VariantProps<typeof StyledNotification>['expand'];
+  color?: VariantProps<typeof StyledNotification>['color'];
+  simple?: VariantProps<typeof StyledNotification>['simple'];
+}
+
+export const NotificationBody = ({
+  children,
+}: {
+  children: React.ReactNode;
+}): JSX.Element => {
+  return <StyledText as="p">{children}</StyledText>;
+};
+
+// For now they are identical, so we can just export a rename of NotificationBody for the Footer
+export const NotificationFooter = NotificationBody;
+
+export const Notification: FC<INotificationProps> = ({
+  icon,
+  title,
+  children,
+  displayCloseButton,
+  color,
+  simple,
+  expand,
+}) => {
+  const Icon = icon!;
+  const isSimple = (simple as boolean) || children === undefined;
+  return (
+    <StyledNotification color={color} expand={expand} simple={isSimple}>
+      <StyledIconContainer position="left">
+        <Icon size="md" />
+      </StyledIconContainer>
+
+      <StyledHeading as="h6">{title}</StyledHeading>
+      {!isSimple && children}
+
+      {displayCloseButton === true && !isSimple && (
```

Comment:
Not something to pick up right away, but I do want to suggest having `header` (red) and `footer` (blue) slots for these components. 

![image](https://github.com/kadena-community/kadena.js/assets/1378630/34da664f-369d-4286-a1c0-dcbf30676ed8)

This way, we can inject more functionalities into the components. In this case, we could have multiple header or footer actions.

Reply:
Noted! Will add to the vanilla-extract rewrite PR